wazo_calld/plugins/conferences/bus_consume.py

```diff
@@ -28,6 +29,12 @@ def subscribe(self, bus_consumer):
 
     def _notify_participant_joined(self, event):
         conference_id = int(event['Conference'])
+        try:
+            conference = Conference.from_id(conference_id, self._confd)
```

Comment:
Would it be useful to add tests for this?

Reply:
:-1: for integration tests, we should not add test if there is no functional impact and be careful with what we tests. Testing too much useless things can be worst than testing nothing
But no objection for unittests

Reply:
Agreed for functional test. I don't like testing for logs, and testing for a non-event is too slow because you have to wait for the timeout. I asked myself the same question for unit-tests, so I guess it's worth it :) I'll add the test.

Reply:
For this fix, I prefer to no have test than blocking the PR while 7 months :shrug: